contrib/hx/template.py

```diff
@@ -0,0 +1,25 @@
+pkgname = "hx"
+pkgver = "1.0.14"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_build_args = [
+    f"hx_git_hash=chimera-{pkgver}-r{pkgrel}",
+    f"hx_version={pkgver}",
+]
+hostmakedepends = ["gmake"]
+pkgdesc = "Terminal Hex Editor"
+maintainer = "ttyyls <contact@behri.org>"
+license = "MIT"
+url = "https://github.com/krpors/hx"
+source = f"https://github.com/krpors/hx/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "7b7c494df149535f768b3aa449159928aec92b468a17f9eedebcea3dd21343d1"
+hardening = ["vis", "cfi"]
+# no tests defined
+options = ["!check"]
+
+
+def do_install(self):
```

Comment:
use install target

Reply:
the install target would still need to manually rename the files after (or patch the makefile), and also uncompress the precompressed manpage. if anything it's more effort than installing 3 files here

Reply:
the makefile needs patching regardless to fix other things so

Reply:
no incompressing is necessary, you just install one instead of the other